test/test.dart

```diff
@@ -194,4 +196,34 @@ Future<void> setupBaas() async {
   final client = await (cluster == null ? BaasClient.docker(baasUrl) : BaasClient.atlas(baasUrl, cluster, apiKey!, privateApiKey!, projectId!));
 
   baasApps.addAll(await client.getOrCreateApps());
-}
\ No newline at end of file
+}
+
+@isTest
+Future<void> syncTest(
```

Comment:
that's an implementation detail in that name. This why I think syncTest is better

Reply:
baas is intentionally called like that to be a generic name unrelated to the marketing name of the product. Using `testWithBaas` or `baasTest` if you prefer a shorter name is better because it covers all the backend functionality, not just sync. Auth, function, and remote mongo tests will not use anything sync-related, but will need to run against a baas instance.

Reply:
lets go with baasTest

Reply:
I'd strongly recommend keeping the baas terminology for consistency's sake. Both the actions that spin up a cluster are using baas as terminology, as well as the dart client that configures the app is called BaasClient. Additionally, the repo of the server is called baas and all SDKs are (should) be using baas when referring to the server. There's no reason to be inconsistent about it.